Scripts - PCB/SelectComponents/SelectComponentScript.pas

```diff
@@ -0,0 +1,86 @@
+Procedure StartScript ();
+var
+   Board              : IPCB_Board;         //���������� ������� �������� �����
```

Comment:
I don't understand what `Ïåðåìåííàÿ îáúåêòà ïå÷àòíîé ïëàòû` means.
Is there a character encoding issue?

Reply:
It's simple, this comment is made in Russian. I myself am from Russia and plan to make instructional videos in Russian.

Reply:
![image](https://user-images.githubusercontent.com/10257640/77741178-0bbac580-7047-11ea-98b8-e043d81e0b1c.png)
Here is how it displays with me

Reply:
Okay, so:
1. Let's try to set the encoding of your files to [UTF-8](https://en.wikipedia.org/wiki/Cyrillic_script_in_Unicode), and commit/push again
1. Use english comments and variable/function names as much as possible

Reply:
How can I do it?
Need to withdraw a request, make changes and create a new request?

Reply:
No, please **do not create a new pull request**, just do as I said.

1. Check the encoding of your file, and if necessary, change it to UTF-8.
For example if you use Notepad++ :
![change encoding](https://user-images.githubusercontent.com/11060755/77750761-a950cd80-7024-11ea-8e23-346c78168f46.png)
1. Save you newly encoded file
1. Commit
1. Push

The pull request will automatically update with your latests modifications.

Reply:
![image](https://user-images.githubusercontent.com/10257640/77756398-a88a5c80-7061-11ea-953f-d7c8ed39f638.png)
I tried changing the encoding. Github in this encoding correctly displays information. But the default Altium designer does not work in this encoding, and incorrectly displays Russian text in the UTF-8 encoding.

Reply:
Apparently the most correct solution is to translate the comments into English.
I'll do it now.

Reply:
I have done.

Reply:
:+1: